exec/java-exec/src/main/java/org/apache/drill/exec/testing/CountDownLatchInjectionImpl.java

```diff
@@ -41,8 +41,11 @@
   private CountDownLatchInjectionImpl(@JsonProperty("address") final String address,
                                       @JsonProperty("port") final int port,
                                       @JsonProperty("siteClass") final String siteClass,
-                                      @JsonProperty("desc") final String desc) throws InjectionConfigurationException {
-    super(address, port, siteClass, desc, 0, 1);
+                                      @JsonProperty("desc") final String desc,
+                                      @JsonProperty("nSkip") final int nSkip,
+                                      @JsonProperty("nFire") final int nFire,
+                                      @JsonProperty("msPause") final Long msPause) throws InjectionConfigurationException {
```

Comment:
long instead of Long? You will save a lot of unnecessary checking for null

Reply:
Might have hit it accidentally. Don't think I need a null check. Will walk through this once and update it.